sky/engine/core/rendering/RenderView.cpp

```diff
@@ -60,7 +59,6 @@ bool RenderView::hitTest(const HitTestRequest& request, HitTestResult& result)
 
 bool RenderView::hitTest(const HitTestRequest& request, const HitTestLocation& location, HitTestResult& result)
 {
-    TRACE_EVENT0("blink", "RenderView::hitTest");
```

Comment:
you have another TRACE_EVENT in glue, right?

wait you have a hitTest function in C++?